compiler/frontend/src/org/jetbrains/kotlin/resolve/calls/model/MutableDataFlowInfoForArguments.java

```diff
@@ -47,6 +48,9 @@ public void updateInfo(@NotNull ValueArgument valueArgument, @NotNull DataFlowIn
             throw new IllegalStateException();
         }
 
+        @Override
+        public void updateResultInfo(@NotNull DataFlowInfo dataFlowInfo) { }
```

Comment:
This is actually reachable during resolution of calls with callable references. Though, it currently reachable only with `EMPTY` info, so it can be fixed with additional check in `CallCompleter.updateResultDataFlowInfoUsingEffects`. Nevertheless, I want to discuss this with @erokhins later.